src/clj/puppetlabs/http/client/common.clj

```diff
@@ -70,7 +74,8 @@
    (ok :body)             Body
    (ok :decompress-body)  schema/Bool
    (ok :as)               BodyType
-   (ok :query-params)     {schema/Str schema/Str}})
+   (ok :query-params)     {schema/Str schema/Str}
+   (ok :metric-id)        [schema/Str]})
```

Comment:
I think it would be nice if the Clojure API tolerated keywords here in addition to Strings.  The Java API doesn't need to.